en/mulesoft/modules/ROOT/pages/ms_composer_invocable_flows.adoc

```diff
@@ -0,0 +1,263 @@
+
+=== Invoke a Composer Flow from Another Composer Flow 
+
+You can create an invocable flow in Composer and invoke it from another Composer flow.
+
+To invoke a Composer flow from another Composer flow:
+
+. Access Composer.
+. xref:ms_composer_flows.adoc#create-a-new-flow[Create a new flow from scratch].
+. Click *+* and click *System Action*. Select *HTTP*.
+. Configure your HTTP connection. For more information about configuring your HTTP connection, refer to HTTP Connector xref:ms_composer_http_reference.adoc#connections[Connections]. 
++
+Ensure the following fields have these values:
+
+* *Authorization Type*: `Basic Authentication`
+* *Base URL*: The URL of the invocable flow
+* *Username*: Client ID of the invocable flow
+* *Password*: Client secret of the invocable flow
+
+. Click *Next*.
+. Configure the fields to test your HTTP connection. For more information about testing your HTTP connection, refer to xref:ms_composer_http_reference.adoc#test-the-connection[Test the Connection]. 
++
+Ensure the following fields have these values:
+
+* *Request Method*: `POST`
+* *Request Path*: `/`
+* *Request Headers*
+** *Key*: `Content-Type`
+** *Value*: `application/json`
+* *Request Body*: Input fields of the invocable flow
+
+. Click *Connect*. After the connection is created, a flow can invoke another invocable flow. 
+
+=== Invoke a Composer Flow from Flow Builder
+
+Invoke a Composer flow directly from Flow Builder to build integrations without writing a single line of code.
+
+For example, use invocable flows to build integrations in Flow Builder that connect to third-party systems such as NetSuite, Jira, and Workday. Some use cases include using Flow Builder to call an invocable Composer flow that:
+
+* Creates a Jira issue when a record is created in Salesforce.
+* Retrieves an order status from NetSuite and posts a message in Slack.
+* Gets real-time credit scores when a user goes through a screen flow.
+* Retrieves an order status from NetSuite when a user enters an order ID in the screen flow.
+
+To invoke a Composer flow from Flow Builder, you must:
+
+. <<set-up-salesforce-for-invocable-flows,Set up Salesforce for invocable flows>>.
+. <<set-up-flow-builder-for-invocable-flows,Set up Flow Builder for invocable flows>>.
+
+[[set-up-salesforce-for-invocable-flows]]
+==== Set Up Salesforce for Invocable Flows
+
+After creating your Composer flow, set up your Salesforce org to invoke flows in Composer: 
+
+. Access your Salesforce org. 
+. Go to *Setup* > *MuleSoft* > *Services* and click *Log In* to securely connect Anypoint Platform with your Salesforce org and grant access.
+. Create a new set of credentials that use the same credentials that you use for Composer. Create an external credential and a named credential. If your invocable Composer flow does not contain input fields, you must add a new `Content-Length` custom header that has a value of `0` in the external credential and named credential to avoid an HTTP 411 response. 
```

Comment:
```suggestion
. Create a new set of credentials that use the same credentials that you use for Composer. 
+
Create an external credential and a named credential. If your invocable Composer flow does not contain input fields, add a new `Content-Length` custom header that has a value of `0` in the external credential and named credential to avoid an HTTP 411 response. 
```